src/easylink/step.py

```diff
@@ -1187,16 +1218,18 @@ class ChoiceStep(Step):
     :attr:`Step.config_key` in the pipeline specification file. Instead, the pipeline
     configuration must contain a 'type' key that specifies which option to choose.
 
+    A ``ChoiceSteps'`` :attr:`choices` dictionary must contain only a *single*
```

Comment:
```suggestion
    The values in a ``ChoiceSteps'`` :attr:`choices` dictionary must each contain only a *single*
```

I am not sure this is the right solution, but this feels a little unclear to me as written -- like it's saying there actually _aren't_ multiple choices.